Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM theconversation/node-ubuntu:16.13.1
+FROM us-docker.pkg.dev/tc-platform-artifacts/theconversation/node:16.14.2
```

Comment:
i'm getting confused...

isn't it node-ubuntu which is 16.14.2?

Reply:
or this dependent on https://github.com/conversation/docker-images/pull/29/files

Reply:
have now merged https://github.com/conversation/docker-images/pull/29 so 👍